config.yaml

```diff
@@ -123,7 +125,20 @@ groot:
     maxlength: 125           # Maxlength for groot index
     covcutoff: 0.97          # Coverage cutoff for groot report
     lowcov: False            # Report ARGs with no 5' or 3' coverage. Overrides covcutoff.
-
+amrplusplus:
+    databases:
+        db: "scripts/amrplusplus/db/megares_modified_database_v2.00.fasta"
+        annotation: "scripts/amrplusplus/db/megares_modified_annotations_v2.00.csv"
+    rarefaction:
+        script: "scripts/amrplusplus/bin/rarefaction"   # path to rarefaction script
```

Comment:
We have not utilized this way of referencing script paths in any previous StaG module. I'm not entirely sure that this belongs in the config file, as usage of the script is tightly coupled to the call to the script (in the rule body), so we might as well just hardcode the script path in there... Thoughts?

Reply:
Sure its easy to change. My main idea with this type of referencing was if we would ever change the structure of `scripts`  folder we would only need to change the path in `config.yaml` and not touch the rulefile itself.

Reply:
I would like to only have stuff that end-users should modify in the config file. You can still use the `params` section to define the path to the script for each rule just like you currently have, just that you hardcode it in place there instead of keeping it here in the config.